### PR TITLE
remove experimental tag for pipes k8s, lamdba, databricks 3/3

### PR DESCRIPTION
**Pull request by yuhan**
## Summary & Motivation
This PR unexperimentalizes:
- Lambda Pipes
  * PipesLambdaClient
  * PipesLambdaLogsMessageReader
  * PipesLambdaEventContextInjector
  * note: PipesS3MessageReader isn't experimental in status quo which might have been an oversight
- K8s Pipes
  * PipesK8sClient
  * PipesK8sPodLogsMessageReader
- Databricks Pipes
  * PipesDatabricksClient
  * PipesDbfsContextInjector
  * PipesDbfsMessageReader
  * PipesDbfsLogReader

### CODE REVIEW

**Comment by yuhan** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/23280?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#23280** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/23280?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#23250** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/23250?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#23249** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/23249?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @yuhan and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: python_modules/libraries/dagster-databricks/dagster_databricks/pipes.py [hunk 1]
```diff
@@ -89,10 +87,14 @@ def get_default_message_reader(self, task: jobs.SubmitTask) -> "PipesDbfsMessage
         if task.as_dict().get("new_cluster", {}).get("cluster_log_conf", {}).get("dbfs", None):
             log_readers = [
                 PipesDbfsLogReader(
-                    client=self.client, remote_log_name="stdout", target_stream=sys.stdout
+                    client=self.client,
```
Comment (schrockn) [inline python_modules/libraries/dagster-databricks/dagster_databricks/pipes.py:90]: unrelated?

**Review by schrockn — APPROVED** [review]
lfg

File: python_modules/libraries/dagster-databricks/dagster_databricks/pipes.py (hunk 1 above)
Comment (yuhan) [reply python_modules/libraries/dagster-databricks/dagster_databricks/pipes.py:90]: yeah but `make ruff` gave me that..

**Comment by yuhan** [comment]
### Merge activity

* **Jul 29, 9:08 AM PDT**: @yuhan started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/dagster-io/dagster/23280).
